src/scripts/breeding/Breeding.ts

```diff
@@ -456,4 +463,35 @@ class Breeding implements Feature {
         }
     }
 
+    private usableQueueSlots = ko.pureComputed(() => {
```

Comment:
It should be public, now that it's used in the html. Works without, but private might make people think that it's not used elsewhere